src/node_crypto.cc

```diff
@@ -3583,7 +3583,7 @@ SignBase::Error Sign::SignFinal(const char* key_pem,
   if (pkey == nullptr)
     goto exit;
 
-  if (EVP_SignFinal(&mdctx_, *sig, sig_len, pkey))
+  if (EVP_SignFinal(&mdctx_, *sig, sig_len, pkey) && 0 == ERR_peek_error())
```

Comment:
Is `ERR_peek_error()` really necessary here? man page says that it can be used to obtain error code, nothing about the indication of error itself.